sentinel-serialization/sentinel-serialization-common/src/main/java/com/alibaba/csp/sentinel/serialization/common/fallback/DateTypeAdapter.java

```diff
@@ -0,0 +1,45 @@
+/*
+ * Copyright 1999-2020 Alibaba Group Holding Ltd.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package com.alibaba.csp.sentinel.serialization.common.fallback;
+
+import java.lang.reflect.Type;
+import java.util.Date;
+
+import com.google.gson.JsonDeserializationContext;
+import com.google.gson.JsonDeserializer;
+import com.google.gson.JsonElement;
+import com.google.gson.JsonParseException;
+import com.google.gson.JsonPrimitive;
+import com.google.gson.JsonSerializationContext;
+import com.google.gson.JsonSerializer;
+
+class DateTypeAdapter implements JsonSerializer<Date>, JsonDeserializer<Date> {
```

Comment:
`GsonDateTypeAdapter` may be a better name, or add separate package for gson and fastjson fallback.